core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -83,7 +82,14 @@
             </div>
           </div>
 
-          <button type="submit" class="btn btn-success protractor-update-form-submit" ng-disabled="!updateFormValues.newRole || !updateFormValues.username" value="update role">Update Role</button>
+          <div class="form-group" ng-if="updateFormValues.newRole==='TOPIC_MANAGER'">
```

Comment:
Just a question. Is this form ready to be released directly? If not please consider hiding it behind ENABLE_NEW_STRUCTURES? 

Not sure. Feel free to ignore the comment if it isn't relevant!

Reply:
I think it's probably fine to just leave this, since it's an admin-only function anyway.